src/generator/AutoRest.Go/rakefile.rb

```diff
@@ -13,18 +16,18 @@
 	compute: {version: "2016-03-30"},
 	# containerservice: {version: "2016-03-30", swagger: "containerservice"},
 	# commerce: {version: "2015-06-01-preview"},
-	# datalake_analytics: {
-	# 	account: {version: "2015-10-01-preview"},
-	# 	catalog: {version: "2015-10-01-preview"},
-	# 	job:  {version: "2016-03-20-preview"}
-	# },
+	datalake_analytics: {
+		account: {version: "2015-10-01-preview"},
+		# catalog: {version: "2016-06-01-preview"},
+		job:  {version: "2016-03-20-preview"}
+	},
 	datalake_store: {
 		account: {version: "2015-10-01-preview"},
 		filesystem: {version: "2015-10-01-preview"}
```

Comment:
These are in arm in [azure-rest-api-specs](https://github.com/Azure/azure-rest-api-specs) and in [.net sdk](https://github.com/Azure/azure-sdk-for-net/tree/AutoRest/src/ResourceManagement/DataLake.Store/Microsoft.Azure.Management.DataLake.Store/Generated) repos.